test-case/check-fw-echo-reference.sh

```diff
@@ -84,17 +75,20 @@ do
         fi
         # run echo reference test
         dlogc "alsabat -P $pb_dev --standalone -c $channel -f $fmt -r $rate -n $frames -F $frequency"
-        alsabat -P "$pb_dev" --standalone -c "$channel" -f "$fmt" -r "$rate" -n "$frames" -F "$frequency" &
+        alsabat -P "$pb_dev" --standalone -c "$channel" -f "$fmt" -r "$rate" -n "$frames" -F "$frequency" & alsabatPID=$!
         # playback may have low latency, add 0.5 second delay to aviod recording zero at beginning.
         sleep 1
         dlogc "alsabat -C $cp_dev -c $channel -f $fmt -r $rate -F $frequency"
-        alsabat -C "$cp_dev" -c "$channel" -f "$fmt" -r "$rate" -F "$frequency"
+        if ! alsabat -C "$cp_dev" -c "$channel" -f "$fmt" -r "$rate" -F "$frequency"; then
+            # upload failed wav files
+            find /tmp -maxdepth 1 -type f -name "bat.wav.*" -size +0 -exec cp {} "$LOG_ROOT/" \;
```

Comment:
ouch... Taking all *.wav from /tmp (up to 1 level deep) seems a bit too broad to me. Can we have a specific directory under /tmp?

Reply:
`mktemp` maybe?

Reply:
The directory that used to store alsabat wav files is defined by the alsabat tool,  all alsabat generated wav files are stored in /tmp with format as bat.wav.xxxxxx, "“xxxxxx”  are random codes.

Reply:
@keqiaozhang oh, have I misread the name pattern? If it's `bat.wav.*` then it isn't that bad any more... But we could also use `--saveplay=` to specify a filename and maybe a directory.

Reply:
@lyakh , right, we can use --saveplay to specify a filename even with a directly, but this is only for playback, so in order to unify the processing of playback and recorded files, I think the default name pattern is easy to handle.